cmd/snap/cmd_snap_op.go

```diff
@@ -97,6 +97,7 @@ var (
 	shortInstallHelp = i18n.G("Install a snap to the system")
 	shortRemoveHelp  = i18n.G("Remove a snap from the system")
 	shortRefreshHelp = i18n.G("Refresh a snap in the system")
+	shortTryHelp     = i18n.G("Try a snap in the system")
```

Comment:
s/a snap/an unpacked snap/

All those "in the system" feel slightly strange.. but let's fix that by itself elsewhere.